src/praisonai-bot/praisonai_bot/gateway/server.py

```
@@ -6156,6 +6161,41 @@ def _stamp_admission_gate(self, bot: Any) -> None:
+        so the recovery is operator-visible.
+        """
+        sess = (
+            getattr(bot, "_session", None)
+            or getattr(bot, "_session_mgr", None)
+        )
+        journal = getattr(sess, "_ingress_journal", None) if sess is not None else None
+        if journal is None:
+            return
+        try:
+            replayed = journal.replay()
+        except Exception as exc:  # pragma: no cover — defensive, recovery is best-effort
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)